src/content/docs/setup/analytics/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,165 @@
+        commerce: true,
+        aep: true
+      }
+    },
+);
+`} lang="js" title="scripts/delayed.js" />
+
+</Task>
+</Tasks>
+
+### Configuration Parameters
+
+| Parameter | Description | Required |
+|-----------|-------------|----------|
+| `aepContext.imsOrgId` | Your Adobe IMS Organization ID | Yes |
+| `aepContext.datastreamId` | Your configured datastream ID for routing data to AEP | Yes |
+| `eventForwardingContext.commerce` | Enables commerce event tracking | Yes |
+| `eventForwardingContext.aep` | Enables forwarding events to Adobe Experience Platform | Yes |
+
+### What This Configuration Does
+
+- **`aepContext`**: Provides the necessary identifiers for routing data to your specific Adobe Experience Platform environment
+- **`eventForwardingContext.aep: true`**: Instructs the Adobe Data Layer to forward commerce events to the Experience Platform Edge Network
+- **`eventForwardingContext.commerce: true`**: Enables tracking of commerce-specific events like product views, cart additions, and purchases
+
+## Data Flow and Events
+
+Once configured, your storefront will automatically send the following types of events to Adobe Experience Platform:
+
+- **Product interactions**: Views, clicks, add-to-cart actions
+- **Shopping behavior**: Cart updates, checkout steps, purchases
+- **User engagement**: Page views, time on site, navigation patterns
+
+These events are processed in real-time and can be used for:
+- Customer journey analysis
+- Real-time personalization
+- Audience segmentation
+- Attribution modeling
+
+## Validation
+
+You can see an example of this <Link href="https://github.com/hlxsites/aem-boilerplate-commerce/commit/eb49d00bcebd21e48cd87c0efced17d2c6ccfd9d" text="in the Commerce Boilerplate" />.
+
+### Testing Your Integration
+
+<Tasks>
+
+<Task>
+### Check browser console
+
+After implementing the configuration, open your browser's developer tools and verify that:
+
+- No JavaScript errors appear
+- Adobe Data Layer events are being fired
+- Network requests to Adobe Experience Platform Edge Network are successful
+</Task>
+
+<Task>
+### Monitor data ingestion
+
+Use Adobe Experience Platform's monitoring tools to confirm data is being received:
+
+- Navigate to your AEP workspace
+- Check the **Monitoring** section for incoming data
+- Verify events appear in your configured datasets
+</Task>
+
+</Tasks>
+
+### Validate event structure
+
+Ensure events contain the expected commerce data fields and customer identifiers
+
+### Troubleshooting
+
+If data is not flowing as expected:
+
+- **Verify credentials**: Double-check your IMS Organization ID and Datastream ID
+- **Check datastream configuration**: Ensure your datastream is properly configured to route to Adobe Experience Platform
+- **Review browser network tab**: Look for failed requests to Adobe Experience Platform endpoints
+- **Validate Adobe Data Layer**: Confirm the Adobe Data Layer is properly initialized before your AEP configuration
```

Comment:
we also may suggest to use AEP Debugger browser extension https://experienceleague.adobe.com/en/docs/experience-platform/debugger/home